Core/GDCore/IDE/WholeProjectRefactorer.h

```diff
@@ -212,6 +212,13 @@ class GD_CORE_API WholeProjectRefactorer {
                                               gd::Object& object,
                                               const gd::String& behaviorType,
                                               const gd::String& behaviorName);
+  /**
+   * \brief Add required behaviors if necessary to fill every behavior
+   * properties of the added behaviors.
```

Comment:
I don't think that the method knows the context in which it is called (a behavior is added) so I would just say:
```suggestion
   * properties of the given behavior.
```